getGit.tex

```diff
@@ -185,361 +215,201 @@ \section{What, Why, and How?}
-                /--o
-               /
-A  B  C     E /  G
-o--o--o--\--o/---o
-          \ 
-           \--o--o--\
-              D  H   \
-                      \--o
-                        DUMB
-\end{verbatim}
+\begin{frame}
+    \vspace{1cm} % generate some space between title and content
+	\begin{columns}
+		\column{2.5in}
+    \frametitle{Our Sample Workflow}
+        1. Pull from Upstream Master to \\
+        Local Master \linebreak\linebreak
+        2. Branch to Working Directory \linebreak\linebreak
+        3. Make and test changes \linebreak\linebreak
+        4. Commit to Local Dev \linebreak\linebreak
+        5. Push to Origin Dev \linebreak\linebreak
+        6. Open pull request to Upstream \\
+        Master
+	\column{2.25in}
+	\begin{center}
+	\includegraphics[width = .9\linewidth]{assets/gitflow1}
+	\end{center}
+	\end{columns}
+    \vspace{1cm} % generate some space between title and content
 \end{frame}
 
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	You develop many features on your projects branches.
-\begin{verbatim}
-                   F
-                /--o
-               /
-A  B  C     E /  G
-o--o--o--\--o/---o
-          \            
-           \--o--o--\--o
-              D  H   \ I
-                      \--o
-                        DUMB
-\end{verbatim}
+\begin{frame}
+    \vspace{1cm} % generate some space between title and content
+	\begin{columns}
+		\column{2.5in}
+    \frametitle{Our Sample Workflow}
+        1. Pull from Upstream Master to \\
+        Local Master \linebreak\linebreak
+        2. Branch to Working Directory \linebreak\linebreak
+        3. Make and test changes \linebreak\linebreak
+        4. Commit to Local Dev \linebreak\linebreak
+        5. Push to Origin Dev \linebreak\linebreak
+        6. Open pull request to Upstream \\
+        Master
+	\column{2.25in}
+	\begin{center}
+	\includegraphics[width = .9\linewidth]{assets/gitflow2}
+	\end{center}
+	\end{columns}
+    \vspace{1cm} % generate some space between title and content
 \end{frame}
 
-\begin{frame}[fragile]
-	\frametitle{Branch: development history}
-	When harvest time comes, you pick the good ones and leave the bad.
-\begin{verbatim}
-                   F
-                /--o
-               /
-A  B  C     E /  G
-o--o--o--\--o/---o
-          \            
-           \--o--o--\--o
-              D  H   \ I
-                      \--o
-                        DUMB
-\end{verbatim}
+\begin{frame}
+    \vspace{1cm} % generate some space between title and content
+	\begin{columns}
+		\column{2.5in}
+    \frametitle{Our Sample Workflow}
+        1. Pull from Upstream Master to \\
+        Local Master \linebreak\linebreak
+        2. Branch to Working Directory \linebreak\linebreak
+        3. Make and test changes \linebreak\linebreak
+        4. Commit to Local Dev \linebreak\linebreak
+        5. Push to Origin Dev \linebreak\linebreak
+        6. Open pull request to Upstream \\
+        Master
```

Comment:
Have you considered animating the application of this list to align with the animation of the flow chart? It would take a little thinking to make sure the steps all line up, but mechanically should be pretty easy.